handlerGithub/github.go

```diff
@@ -58,10 +66,14 @@ type Components struct {
 	Sonarqube sonarqube.Sonarqube `json:"Sonarqube,omitempty"`
 }
 
-func initGitHubClient() *githubApi.Client {
+func initGitHubClient() *githubApi.Client { // TODO return error here or just do fatal?
 	//	if len(appKey) < 1 {
 	//		log.Fatalln("Missing App Key")
 	//	}
+	appID, err := strconv.ParseInt(os.Getenv("appid"), 10, 64)
```

Comment:
Any reason to re-read and parse the env on every init of the gh-client? Why read it once like `repoAsCodeOrg` and the other global vars?

Reply:
Or maybe read `appID` before calling `initGitHubClient`, add pass it as an argument

Reply:
Could setup a function that checks the environment variables and sets them accordingly i guess. might be bad code quality hmm

Reply:
Oh read it in main function, Yeah sure we can do that.

Reply:
Passing it sounds sounds a lot better

Reply:
I will setup an init function that sorts this out. This way we can also check mandatory values :)

Reply:
I played with the idea of creating a function in main that checks all the environment variables but thats bad. I believe that the best practice is either having an init function for the module that checks the environment variables or not even setting vars but instead calling each time.

Reply:
Or we just let the function that needs these variables fail because they will fail when you pass an empty argument i think